distributed/scheduler.py

```diff
@@ -281,6 +281,7 @@ def __init__(
         self.versions = versions or {}
         self.nanny = nanny
 
+        self._hash = hash(address)
```

Comment:
Could you add `"_hash"` to `WorkerState.__slots__`?

Reply:
Thanks James! I completely missed that. Added above 🙂